### ISSUE TITLE
Android/iOS managed configurations

### ISSUE THREAD

**Issue by roberthase** [netbirdio/netbird#1918]
**Is your feature request related to a problem? Please describe.**
manual configuration of selfhosted-servers and no way to provide setup-keys on android/ios

**Describe the solution you'd like**
provide a managed configuration for the netbird app which can be rolled out with a mdm (intune, google workspace)

**Describe alternatives you've considered**
trusting users to read provided directions

**Additional context**
i would try to convince my company to contribute($$$) for the implementation

**Comment by mrbluecoat** [comment]
ref: https://github.com/netbirdio/netbird/issues/1661

**Comment by wwwryanwww** [comment]
I would like to see this feature implemented too.  We would use it to push out a Netbird setup key and configuration to all of our iPads.

**Comment by Lauriumora** [comment]
Yes, please implement this feature. This is the only way in which Netbird can get any traction in enterprise level Android/IOS deployments.
Please see this link on how it should be done on Android https://developer.android.com/work/managed-configurations
Most MDMs will support this.

Note that is ideally accompanied by agent functionality that basically disallows end-users to do anything with the Netbird agent (e.g. company policy enforcement)

**Comment by mcgurrin** [comment]
also looking for this to make deployment and management easier

**Comment by stoimen-modeshift** [comment]
We are operating a fleet of android devices used as validators in public transport busses. Entering the setup key from the UI is not a solution for us. We would like to to have unattended install and configuration of the apk with setup keys.
